demisto_sdk/commands/common/errors.py

```diff
@@ -1532,6 +1535,16 @@ def invalid_file_path_layoutscontainer(file_name):
         return f'Invalid file name - {file_name}. layoutscontainer file name should start with ' \
                '"layoutscontainer-" prefix.'
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_file_path_pre_process_rules(file_name):
+        return f'Invalid file name - {file_name}. Pre Process Rules file name should start with "preprocessrule-" prefix.'
```

Comment:
no need to have this validation

Reply:
https://github.com/demisto/demisto-sdk/commit/7b04b7162cfea19b1b9a3a0e0766415390eca435